{
	"deprecated": false,
	"cpeName": "cpe:2.3:a:sage:syracuse:12.10.2.8:*:*:*:*:*:*:*",
	"cpeNameId": "C14E8BA1-31F5-4485-8F39-917BE07CD103",
	"created": "2021-07-28T12:55:11.823",
	"lastModified": "2021-08-19T18:25:09.330",
	"titles": [
		{
			"title": "Sage Syracuse 12.10.2.8",
			"lang": "en"
		}
	],
	"refs": [
		{
			"ref": "https://github.com/CVEProject/cvelist/blob/master/2020/7xxx/CVE-2020-7389.json",
			"type": "Advisory"
		},
		{
			"ref": "https://www.rapid7.com/blog/post/2021/07/07/cve-2020-7387-7390-multiple-sage-x3-vulnerabilities/",
			"type": "Advisory"
		},
		{
			"ref": "https://support.na.sage.com/selfservice/viewContent.do?externalId=109778&sliceId=1&isAuthoredDocumenttrue&accessCount=177&positiveRatingCount=5",
			"type": "Advisory"
		},
		{
			"ref": "https://www.sage.com/en-gb/company/",
			"type": "Vendor"
		}
	]
}
